ios/RNWifi.m

```diff
@@ -123,6 +123,10 @@ + (BOOL)requiresMainQueueSetup
                   rejecter:(RCTPromiseRejectBlock)reject) {
     // Prevent NEHotspotConfigurationManager error when connecting to an already connected network
     if ([ssid isEqualToString:[self getWifiSSID]]) resolve(nil);
+    if ([ssid isEqualToString:[self getWifiSSID]]) {
+        resolve(nil);
```

Comment:
Duplicate of https://github.com/JuanSeBestia/react-native-wifi-reborn/pull/103#discussion_r470192478